docs/oscar_references.bib

```diff
@@ -1,7 +1,8 @@
-@article{GDF20,
+@misc{GDF20,
```

Comment:
I would also do this (to match the bibkey the BibTeX style `alpha` would produce):
```suggestion
@misc{DF20,
```

Reply:
Yes.
If we want our keys to be consistent with the `alpha` style then four more keys have to be adjusted.
I have updated the pull request accordingly.